cmd/skenario/main.go

```diff
@@ -16,19 +16,259 @@
 package main
 
 import (
+	"bytes"
```

Comment:
As discussed, let's preserve small main.go from pivotal/master

Reply:
Fixed it. See commit 7fb9b1bb4ec4e399844d7d9b23bf6eeb59de0c06.